task_hive/.github/workflows/sonarcloud.yml

```
@@ -0,0 +1,40 @@
+        uses: actions/checkout@v3
+
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+
+      - name: Setup Flutter
+        uses: subosito/flutter-action@v2
+        with:
+          flutter-version: '3.24.4' 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)